coalib/bears/requirements/DistributionRequirement.py

```diff
@@ -1,4 +1,9 @@
-import platform
+import subprocess
+
+try:
+    import distro
+except subprocess.CalledProcessError:  # pragma: no cover
+    pass
```

Comment:
There are many ways:
 - Check if distro is in locals() or globals()
 - Use sys to check if a module named distro has been loaded
 - Create a flag which is a different value if this is loaded, and another value if it is not loaded
 - Use if conditions with platform.system()
 - Use a try catch with the rest of the distro code and suppress NameErrors because distro is not found